LICENSE.txt

```diff
@@ -0,0 +1,137 @@
+OPEN SOURCE LICENSE AGREEMENT
```

Comment:
We would like this to be LICENSE.rst, but rst format but with the text from LICENSE.txt

Reply:
done